internal/curator/durable/state/state.go

```diff
@@ -462,15 +462,22 @@ outer:
 }
 
 // PutPartition puts a partition into state.
-func (t *Txn) PutPartition(partition *fb.Partition) {
-	t.put(partitionBucket, partitionID2Key(core.PartitionID(partition.Id)), fb.BuildPartition(partition), defaultFillPct)
+func (t *Txn) PutPartition(id core.PartitionID, partition *fb.Partition) {
+	t.put(partitionBucket, partitionID2Key(id), fb.BuildPartition(partition), defaultFillPct)
+}
+
+type PartitionAndID struct {
```

Comment:
nit: should this name follow conventions in the fb docs.go, and use PartitionFAndID? I see why we would drop the `F` cause it reads strange, but we are carrying the fb type here right?

Reply:
It's definitely the fb type. I guess that would be technically correct but it does look odd and is annoying to type. I don't really see any potential confusion here, since there is no struct that has a core.PartitionID and a fb.Partition (not F). I'll leave it like this unless you feel strongly.

Reply:
no strong feelings about it.